Elmah.Io.ElasticSearch/ElasticClientSingleton.cs

```diff
@@ -33,9 +37,9 @@ private ElasticClientSingleton()
             }
         }
 
-        public static ElasticClientSingleton Instance
+        public static ElasticClientSingleton GetInstance(IDictionary config)

```

Comment:
@jayhilden What about this change? Are you cool with this? There could be someone that is using the Instance property, and this will break their code.